src/main/java/net/tropicraft/core/common/entity/underdasea/atlantoku/EntityTropicraftWaterBase.java

```diff
@@ -222,9 +238,10 @@ public void onLivingUpdate() {
 			if(this.fleeFromPlayers) {
 				EntityPlayer closest = world.getClosestPlayerToEntity(this, this.fleeDistance);
 				if(closest != null) {
-					if(closest.isInWater()) 
-						this.fleeEntity(closest);
-						this.isPanicking = true;
+					if(closest.isInWater()) {
+					//	this.fleeEntity(closest);
```

Comment:
no more flee?

Reply:
fleeing the player that was was for debug purposes, i will be creating lists of types of entities to flee from, as well as lists for entities to snack on